adam-core/src/main/scala/org/bdgenomics/adam/converters/VariantContextConverter.scala

```diff
@@ -26,7 +26,11 @@ import htsjdk.variant.variantcontext.{
 }
 import java.util.Collections
 import org.apache.spark.Logging
-import org.bdgenomics.adam.models.{ VariantContext => ADAMVariantContext, SequenceDictionary }
+import org.bdgenomics.adam.models.{
```

Comment:
for what it's worth, I'd be ok bumping the line width in the formatter to 120 characters :)